packages/mdxts/package.json

```diff
@@ -132,7 +132,7 @@
     "hast-util-to-string": "^3.0.0",
     "mdast-util-to-string": "^4.0.0",
     "minimatch": "^9.0.4",
-    "read-package-up": "^11.0.0",
+    "read-pkg-up": "^7.0.1",
```

Comment:
Downgrading `read-package-up` package to utilize CJS version to keep `getAllData` synchronous.